.evergreen/start-atlas-cloud-cluster.sh

```diff
@@ -46,6 +46,10 @@
 #
 #   (ATLAS_CLOUD_TEST_CLUSTER_NAME="TestCluster" source .evergreen/start-atlas-cloud-cluster.sh \
 #     && npm run -w compass-e2e-tests test web -- --test-atlas-cloud-sandbox --test-filter="atlas-cloud/**/*")
+#
+# When setting up for the first time, make sure you:
+# - Add payment details to be able to create clusters. You can use stripe test card.
```

Comment:
Thanks for updating the documentation here! Can you move this to a more logical place where it would fit if you're doing these actions step by step? Probably either as part of creating a new org / project or as a step after it. The intention for the whole instruction above to be about setting up for the first time, so adding it as notes at the bottom of that is a bit weird 🙂

Reply:
yeah will do that!